src/aleph/model/messages.py

```diff
@@ -139,17 +144,20 @@ async def get_computed_address_aggregates(address_list=None, key_list=None, limi
         {"$addFields": {"address": "$_id", "contents": {"$arrayToObject": "$items"}}},
         {"$project": {"_id": 0, "address": 1, "contents": 1}},
     ]
+
+    # Note: type-ignore statements are required because mypy does not understand that
+    # the nested dictionaries we declared just above are dictionaries.
```

Comment:
Could this be fixed with type annotation on the variable ?

Reply:
Technically yes, with [TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict). But it's gonna be a PITA to maintain so I'd rather keep the type ignores. Also, there's an off-chance that this will someday be fixed in mypy. That day, the type ignore statements will trigger errors so we will know :stuck_out_tongue: